compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -2401,6 +2618,13 @@ static LogicalResult setRootConfig(IREE::GPU::TargetAttr target,
       LDBG("Contract Config");
       return success();
     }
+    if (clGPUTestVectorDistributeOnReduction) {
+      if (succeeded(setReductionVectorDistributionConfig(target, entryPointFn,
+                                                         linalgOp))) {
+        LDBG("Vector distribute on  Generics with right config propagation.");
+        return success();
+      }
+    }
```

Comment:
nit: better error message would be: "Vector Distribution Subgroup Reduction Config"